src/pykeen/datasets/base.py

```diff
@@ -431,9 +430,9 @@ class RemoteDataset(PathDataset):
     def __init__(
         self,
         url: str,
-        relative_training_path: str,
-        relative_testing_path: str,
-        relative_validation_path: str,
+        relative_training_path: pathlib.PurePath,
```

Comment:
this one should allow for strings as well. If they are strings, immediately parse them into `pathlib.PurePaths` in the `__init__`